Packages/packages-lock.json

```diff
@@ -16,6 +9,22 @@
       },
       "url": "https://packages.unity.com"
     },
+    "com.unity.ext.nunit": {
+      "version": "1.0.6",
+      "depth": 1,
+      "source": "registry",
+      "dependencies": {},
+      "url": "https://packages.unity.com"
+    },
+    "com.unity.ide.rider": {
```

Comment:
we should remove these packages

Reply:
these packages = rider yah?

Reply:
rider and nunit -- it seems nunit only got installed bc of rider.

correct me if im wrong on that tho